cypress/support/constants/constants.js

```diff
@@ -91,6 +91,7 @@ module.exports = {
   EMAIL: 'email',
   ENV_PLATFORM_SDK_VERSION: 'platformSdkVersion',
   ERROR: 'error',
+  ERROR_CODE: 'Error code',
```

Comment:
Is this being used? I don't see any reference to it in this PR.
If not used, please remove.

Reply:
yes, it is used in 
`            fireLog.include(
              errorContentObject.errorCode,
              apiErrorResponse.code,
              CONSTANTS.ERROR_CODE
            );
`
that's why we had undefined error, ERROR_CODE wasn't added to constants